packages/browser/test/e2e/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "sentry-browser-integration-test-app",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "engines": {
+    "node": ">=12"
+  },
+  "private": true,
+  "babel": {
+    "presets": [
+      "@babel/preset-env"
+    ]
+  },
+  "scripts": {
+    "postinstall": "playwright install --with-deps",
+    "pretest": "yarn install && webpack",
+    "test": "playwright test ./test --browser=all",
+    "posttest": "rimraf dist",
+    "dev": "webpack-dev-server"
+  },
+  "dependencies": {
+    "@babel/core": "^7.15.5",
+    "@babel/preset-env": "^7.15.4",
+    "@playwright/test": "1.17.0-next-alpha-nov-5-2021",
```

Comment:
Why this version?

Reply:
Switched to this due to https://github.com/microsoft/playwright/issues/9811, it'll probably be shipped with the next release in a few days.